### ISSUE TITLE
Add native support for Tiles and freemarker based view templates alongside support for JSP and Facelets [SWF-590]

### ISSUE THREAD

**Issue by spring-operator** [spring-projects/spring-webflow#215]
**[Keith Donald](https://jira.spring.io/secure/ViewProfile.jspa?name=kdonald)** opened **[SWF-590](https://jira.spring.io/browse/SWF-590?redirect=false)** and commented

Currently convention-based view resolution resolves to jsps in a MVC environment.  This is nice.

However, tiles integration requires special configuration and does not easily work with views inside a flow working directory - you resort to a custom view resolver.  Velocity and Freemarker are usable, but also require special configuration and view resolvers.

Basically, it should be as easy to use all these technologies as easy it is to use plain JSP and JSF in SWF today.

---

**Affects:** 2.0.0.RC1

2 votes, 3 watchers
